packages/frontend/src/components/DashboardTiles/EmptyStateNoTiles/index.tsx

```diff
@@ -16,10 +17,7 @@ const SavedChartsAvailable = () => (
     <EmptyStateWrapper>
         <EmptyStateIcon icon="grouped-bar-chart" size={59} />
         <Title>Start building your dashboard!</Title>
-        <p>
-            Click ‘Edit dashboard’ to start adding charts and tiles. Don’t
-            forget to hit ‘Save’ when you’re done!
-        </p>
+        <AddTileButton onAddTiles={() => {}} />
```

Comment:
do you need a onAddTiles prop here? if its optional you can mark it inside `AddTileButton` as optional